packages/v4/src/content/design-guidelines/components/menu/menu.md

```diff
@@ -0,0 +1,82 @@
+
+## Elements and variations
+### Action menu
+An actions menu is used for presenting a list of actions or links.
+
+<img src="./img/basic-menu.png" alt="actions dropdown" width="863"/>
+
+1. **Menu item:** Menu items represent discrete actions that can be taken. Selecting a menu item inside a dropdown will trigger the action and dismiss the menu, while selecting a menu item inside a select list will select the item.
+
+2. **Icon (optional):** You may place an icon before each menu item to aid in recognition. It has been shown that familiar icons speed the recognition of text labels, but only use icons if they will be easily recognized and are distinguishable. Never use icons simply as a decoration.
+
+3. **Links:** Links may be used in place of or mixed with actions to navigate the user to a new page. If the link will open in a new window, the external link icon should be used to annotate the link and let the user know to expect this behavior.
+
+4. **Descriptions (optional):** Descriptive text may be added below each menu item. This should only be used when the menu item label by itself may not be clear to all users. Keep descriptive text to two lines or less.
+
+### Option select menus
+Use an option select menu when you want to persist the selected items. This is typically the case for select menus or settings menus that allow users to select between multiple options. See the [Select](/components/select) and [Options menu](/components/options-menu) components for more details. You may present a single or multiple groups of options within the same menu.
+
+<img src="./img/option-select-menu.png" alt="option select menus" width="863"/>
+
+1. **Selected item:** A checkmark to the right of a menu item indicates that it is currently selected.
+
+2. **Icon (optional):** You may place an icon before each menu item to aid in recognition. It has been shown that familiar icons speed the recognition of text labels, but only use icons if they will be easily recognized and are distinguishable. Never use icons simply as a decoration.
+
+3. **Descriptions (optional):** Descriptive text may be added below each menu item. This should only be used when the menu item label by itself may not be clear to all users. Keep descriptive text to two lines or less.
+
+### Grouped menus
+You may decide to group menu items to associate related items and/or to indicate a hierarchy of items. Items may be grouped using group headings, separators, or both.
+
+<img src="./img/grouped-menu.png" alt="grouped menu" width="348"/>
+
+1. **Group heading (optional):** Add a group heading when you want to name the group.
+
+2. **Separator:** Separators are horizontal dividers that help to group the menu items by clearly showing where one group ends and the next begins.
```

Comment:
@mcarrano Do you disagree with this^?

Reply:
I can change this to optional @mmenestr . I guess it raises a question of when to use a separator and when not to. I looked at the examples for Select. I think the grouped checkbox select examples are fine without separators, but for other examples it would be hard to see the grouping without them. I think this is partly because the style of the section headers is not that distinct from other menu items. Thoughts?

Reply:
That's a good point, maybe we say that in dropdown menus it makes sense to use a separator where as in select lists it's not necessary because there is more visual distinction!